cmd/krew/cmd/info.go

```diff
@@ -33,14 +33,17 @@ var infoCmd = &cobra.Command{
 	Short: "Info shows plugin details",
 	Long: `Info shows plugin details.
 Use this command to find out about plugin requirements and caveats.`,
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
 		for _, arg := range args {
 			plugin, err := indexscanner.LoadPluginFileFromFS(paths.Index, arg)
 			if err != nil {
-				glog.Fatal(err)
+				glog.V(4).Info(err.Error())
```

Comment:
I think we can keep `Run` as opposed to `RunE` for consistency right now. I recall we had an issue with RunE that was causing a user-facing experience issue, so please revert that part.

I think we can do something like:

```
if os.IsNotExist(err) {
    glog.Fatal("plugin not found")
} else if err != nil {
    glog.Fatal("err")
}
```

Reply:
`LoadPluginListFromFS` function returns wrapped error, so we cannot use `os.IsNotExist`.

```go
// LoadPluginListFromFS will parse and retrieve all plugin files.
func LoadPluginListFromFS(indexDir string) (index.IndexList, error) {
	var indexList index.IndexList
	indexDir, err := filepath.EvalSymlinks(indexDir)
	if err != nil {
		return indexList, err
	}

	files, err := ioutil.ReadDir(filepath.Join(indexDir, "plugins"))
	if err != nil {
		return indexList, fmt.Errorf("failed to open index dir, err: %v", err)
	}
```

I will change `LoadPluginListFromFS` function to return non wrapped error. WDYT?

Reply:
Sorry, please ignore my earlier comment. 

I think the safest option here would be to do return an unwrapped error from `ReadPluginFile` only when the error is `os.IsNotExist`.

```
func ReadPluginFile(indexFilePath string) (index.Plugin, error) {
	f, err := os.Open(indexFilePath)
	if err != nil {
		return index.Plugin{}, fmt.Errorf("failed to open index file, err: %v", err)
	}
	return DecodePluginFile(f)
}
```

I think if you do it like:

```
if os.IsNotExist(err) { return err }
else if err != nil { return wrapped err }
```

and if you document that `...when plugin file not found, it returns an error that can be checked with os.IsNotExist.` we can solve this easily.

+ Need to write unit tests for those.

Reply:
I fixed at 741f17fd915edb1aae49845e84ba04456bf832c6.